compiler/rustc_error_codes/src/error_codes/E0789.md

```diff
@@ -0,0 +1,28 @@
+#### This error code is internal to the compiler and will not be emitted with normal Rust code.
+
+The internal `rustc_allowed_through_unstable_modules` attribute must be used
+on an item with a `stable` attribute.
+
```

Comment:
Still missing the style formatting "Erroneous code example". ;)

Reply:
Oops. Won't it be nice when tidy can finally lint this properly!

Reply:
I'm definitely looking forward to it!